main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/gin-gonic/gin"
+	_ "github.com/go-sql-driver/mysql"
+	"golang.org/x/crypto/bcrypt"
+)
+
+var db *sql.DB
+
+func main() {
+	// Open MySQL connection
+	var err error
+	db, err = sql.Open("mysql", "root:ayushsql@tcp(127.0.0.1:3306)/authDB")
```

Comment:
Put username, password, db and url in another file

Take in an environment variable and if env == local then use these variables

Reply:
Got it.